pipelines/jobs/configurations/jdk11u.groovy

```diff
@@ -1,7 +1,6 @@
 targetConfigurations = [
         'x64Mac'        : [    'temurin',    'openj9'                    ],
         'x64Linux'      : [    'temurin',    'openj9',    'dragonwell',    'corretto',    'bisheng',    'fast_startup'],
-        'x64AlpineLinux': [    'temurin'                            ],
```

Comment:
wrong platform? This PR says it moves the aarch64 alpine build?

Reply:
yes, my bad, will fix it